### ISSUE TITLE
BP011: replace "unitary part" with "part core"

### ISSUE THREAD

**Issue by jakebeal** [SynBioDex/SBOL-examples#32]
In [BP011 - Representation of Parts and Devices for Build Planning](https://github.com/SynBioDex/SBOL-examples/tree/main/SBOL/best-practices/BP011), we used the term "unitary part" to describe a design that doesn't have any information about how it is intended to be assembled or otherwise composed with other parts, such as a minimal CDS or promoter sequence.

In our more recent [Functional Synthetic Biology article](https://academic.oup.com/synbio/article/8/1/ysad006/7111205), however, we found that it made sense to reserve the term "part" for designs that _do_ have an "build interface." Subsequent discussions have led to the conclusion that the term "unitary part" is not very descriptive, so after debating a number of alternatives, we propose to change the term to "part core."

The new "part core" term still carries the idea that the design is the central notion of the part, but makes it clear that there may be something missing that is required for it to be complete.

Before we make a pull request for this change, however, we'd like to see if there is general consensus that this change is a good one to make. Please thumbs up/down and/or comment with opinions for or against this proposal.

**Comment by Gonza10V** [comment]
I agree, the suggested change is more informative. In order for a DNA sequence to be a part it shoud be in the context of the whole, like a part of what? As you discussed that "part" may be the problem there, what other alternatives do we have and why you choosed to maintain "part" on the name? 
What about DNA_fragment ? It does not impose any connotation nor role to the object. Then it can be transformed into a "part insert" by adding interfaces. What is the conceptual difference between a DNA fragment and a "part core"
Maybe a con for DNA_fragment is that a backbone falls in that category as well.

**Comment by mburridge96** [comment]
So if we take a GFP sequence, would that mean that the "part core" would be the sequence encoding GFP, whereas the "part" would be that same GFP sequence, but with phytobrick assembly prefix/suffix "interface" sites for example?

**Comment by jakebeal** [comment]
@Gonza10V I proposed "fragment" earlier, but @GC-repeat observed that keeping some sort of relationship to "part" in the name to keep the relationship clear. I agreed with his suggestion.

@mburridge96 That is exactly correct: the GFP sequence itself is "part core" and adding the suffixes would make it the "part insert", then put it into pSB1C5 to make "part-in-backbone", etc.

**Comment by rkrishnasanka** [comment]
Another alternative would be `part payload`
